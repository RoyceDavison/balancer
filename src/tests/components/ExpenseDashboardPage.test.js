import { shallow } from "enzyme"; //这个是需要config setUp的
import React from "react";
import ExpenseDashboardPage from "../../components/ExpenseDashboardPage";

test("Airbnb_enzyme: should render ExpenseDashboardPage properly", () => {
  const wrapper = shallow(<ExpenseDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
