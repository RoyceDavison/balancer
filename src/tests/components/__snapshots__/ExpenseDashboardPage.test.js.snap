// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Airbnb_enzyme: should render ExpenseDashboardPage properly 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
